recipes/libgcrypt/libgcrypt.inc

```diff
@@ -23,13 +23,6 @@ EXTRA_OECONF = "--without-pth --disable-asm ${EXTRA_OECONF_CAPABILITIES}"
 EXTRA_OECONF_CAPABILITIES = "--without-capabilities"
 EXTRA_OECONF_CAPABILITIES:USE_libgcrypt_capabilities = "--with-capabilities"
 
-# libgcrypt.pc is added locally and thus installed here
```

Comment:
Yes, the code removed here is from OE [1], I checked debian, ubuntu and arch and they do not handle the .pc.in file and do therefor not have the libgcrypt.pc file in their dev packages [2-4]. Non of the other gnupg libraries have a .pc file. We might benefit from Not installing/depending on an apparently unused package config file.

[1] http://cgit.openembedded.org/openembedded-core/tree/meta/recipes-support/libgcrypt
[2] https://www.archlinux.org/packages/core/x86_64/libgcrypt/
[3] https://packages.debian.org/stretch/amd64/libgcrypt20-dev/filelist
[4] http://packages.ubuntu.com/xenial/amd64/libgcrypt20-dev/filelist